src/gui/symbology/qgsstylemanagerdialog.cpp

```diff
@@ -222,6 +223,11 @@ QgsStyleManagerDialog::QgsStyleManagerDialog( QgsStyle *style, QWidget *parent,
     mCopyToDefaultButton->hide();
   }
 
+  mActionCopyItem = new QAction( tr( "Copy Item" ), this );
+  connect( mActionCopyItem, &QAction::triggered, this, &QgsStyleManagerDialog::copyItem );
+  mActionPasteItem = new QAction( tr( "Paste Item…" ), this );
```

Comment:
I think the paste normally has no ellipsis in other places, but prove me wrong

```suggestion
  mActionPasteItem = new QAction( tr( "Paste Item" ), this );
```

Reply:
In this case it opens a dialog asking for symbol name and tags, so an ellipsis is appropriate